docs/guides/lint-asyncapi.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Lint AsyncAPI
```

Comment:
These files are synced and published in the [Redocly CLI documentation](https://redocly.com/docs/cli/), correct? 
If so, adding SEO information here will make this guide more discoverable through search.
Recommendation:
```
---
seo:
  title: Lint AsyncAPI with Redocly CLI
  description: Unlock powerful linting capabilities for AsyncAPI documents. Use the Redocly CLI to enforce basic validation, configure rules, or even build custom plugins for AsyncAPI.
---
```

Reply:
Yes, these docs serve both as local markdown docs in the repo, and we also publish them to our main docs site. We don't have SEO entries on any of our pages, not even the main intended entry points. I've opened #1255 to capture this.